lib/features.sh

```diff
@@ -147,6 +147,22 @@ features_get() {
   fi
 }
 
+features_get_with_blank() {
```

Comment:
Should this be the default behaviour of `features_get` rather than a separate method? (Though even if so, perhaps merging them is best in another PR anyway)

Reply:
I agree - I'll have to do this later though because it broke tests with the other feature and it started to get way out of scope.